src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -209,6 +209,20 @@ private static void init() {}
         public static final TagKey<Block> ORES_QUARTZ = cTag("ores/quartz");
         public static final TagKey<Block> PLAYER_WORKSTATIONS_CRAFTING_TABLES = cTag("player_workstations/crafting_tables");
         public static final TagKey<Block> PLAYER_WORKSTATIONS_FURNACES = cTag("player_workstations/furnaces");
+        public static final TagKey<Block> PUMPKINS = cTag("pumpkins");
+        /**
+         * For pumpkins that are not carved.
+         */
+        public static final TagKey<Block> PUMPKINS_NORMAL = cTag("pumpkins/normal");
+        /**
+         * For pumpkins that are already carved but not a light source.
+         */
+        public static final TagKey<Block> PUMPKINS_CARVED = cTag("pumpkins/carved");
+
+        /**
+         * For pumpkins that are already carved and a light source.
+         */
+        public static final TagKey<Block> PUMPKINS_JACK_O_LANTERNS = cTag("pumpkins/jack_o_lanterns");
```

Comment:
For sake of simplicity, I would prefer if these javadoc comments are single-lined.
```suggestion
        public static final TagKey<Block> PUMPKINS = cTag("pumpkins");
        /** For pumpkins that are not carved. */
        public static final TagKey<Block> PUMPKINS_NORMAL = cTag("pumpkins/normal");
        /** For pumpkins that are already carved but not a light source. */
        public static final TagKey<Block> PUMPKINS_CARVED = cTag("pumpkins/carved");

        /** For pumpkins that are already carved and a light source. */
        public static final TagKey<Block> PUMPKINS_JACK_O_LANTERNS = cTag("pumpkins/jack_o_lanterns");
```